### DISCUSSION TITLE
Wallet has been hacked and lost a lot XCH

### DISCUSSION THREAD

**Discussion by xingzi178** [Chia-Network/chia-blockchain#14843] [Support]
on 5th March my wallet was hacked and lost around 20 XCH. they are sent to the address xch16euvttx9ynd68xzl0qefc6k2jahycuwzr7umlqya4z4hq57r2dkq6tdugu.
is there still anything I can do to get the lost XCH back? really appreciate if someone can help!
these transactions happened at height:

3337372
3335972
3334895
3335492
3335439
3335216
3335189
3335144
3334895
3334613
3334750
3334752
3334752

**Comment by xingzi178** [comment]
![Screenshot 2023-03-17 211551](https://user-images.githubusercontent.com/3358070/226036068-c026ad9a-eb40-492a-9387-e5480faa68b5.png)

**Comment by ciscoeivi** [comment]
我也和你一样，也是2023.3.5-2023.3.6这两天被盗取的，总共被盗取130多枚XCH。也希望有人指点怎么找回，重金感谢！

**Reply by xingzi178** [reply]
你都加过什么矿池？有无早期非官方协议矿池？我高度怀疑是我早期加的矿池盗了我的钱包

**Comment by abenlee** [comment]
Do you use xiangrikui remote control or other remote control?

**Reply by xingzi178** [reply]
I didn't use any remote control, but joined non-official pool before the official pooling protocol released. Do I have any chance to get the stole coins back?

**Comment by ciscoeivi** [comment]
有可能，之前有个copool，加了一个月两个币，还没有给我，说我的没有爆块，必须要暴块了才能分币

xingzi178 ***@***.***> 于 2023年3月19日周日 03:57写道：

**Reply by xingzi178** [reply]
那你现在有任何办法追回吗？报警管用吗

**Comment by hajes** [comment]
so, you have given off your private keys as we used to do at Hpool. And you wonder your wallet has been "hacked". Security 101 ;-)
